librz/socket/run.c

```diff
@@ -1245,13 +1245,12 @@ RZ_API int rz_run_start(RzRunProfile *p) {
 				}
 			}
 			setsid();
-#if !LIBC_HAVE_FORK
+#if !HAVE_FORK
```

Comment:
mmm shouldn't this be 
```suggestion
#if HAVE_EXECV
```
?

Reply:
Ｙｅｓ，　ｇｏｏｄ　ｃａｔｃｈ．